### PR TITLE
Ui sign up

### PR DESCRIPTION
**Pull request by YakoobHammouri**
update Sign  up Ui 
add Validation File

---

Sign Up before 

singup

---

Sign Up after

### CODE REVIEW

**Review by sara219 — APPROVED** [review]

File: client/src/Components/Layout/SignUp/SignUp.jsx [hunk 1]
```diff
@@ -0,0 +1,597 @@
...
+
+  handleClickShowPassword = () => {
+    const showPassword = this.state.showPassword;
+
+    this.setState({
+      showPassword: !showPassword,
+    });
+  };
+
+  handleMouseDownRePassword = (event) => {
```
Comment (okjake) [inline client/src/Components/Layout/SignUp/SignUp.jsx:132]: I'm curious, what are these mousedown handlers for?

File: client/src/Components/Layout/SignUp/SignUp.jsx [hunk 2]
```diff
@@ -0,0 +1,597 @@
...
+                          <Input
+                            id="rePassword"
+                            type={showRePassword ? 'text' : 'password'}
+                            value={rePassword.value}
+                            error={!rePassword.isValid}
+                            onChange={this.handleTextInput}
+                            required={true}
+                            name="rePassword"
+                            color="secondary"
+                            label="Enter your re-password"
```
Comment (okjake) [inline client/src/Components/Layout/SignUp/SignUp.jsx:494]: Re-enter your password

**Review by okjake — APPROVED** [review]

File: client/src/Components/Layout/SignUp/SignUp.jsx (hunk 1 above)
Comment (YakoobHammouri) [reply client/src/Components/Layout/SignUp/SignUp.jsx:132]: it for icon in password field

File: client/src/Components/Layout/SignUp/SignUp.jsx (hunk 2 above)
Comment (YakoobHammouri) [reply client/src/Components/Layout/SignUp/SignUp.jsx:494]: done

**Pull request merged**